src/nikhilMJun20/MaxValueAtDiffIndex.java

```diff
@@ -0,0 +1,28 @@
+/*Find max value at differed index.
+i/p:- arr1: {10,2,9,14,3}
+arr2: {10,2,18,14,3}
+o/p :- Values are not matching at index -> 2, From (9,18) max value is 18
+*/
+
+package nikhilMJun20;
+
+public class MaxValueAtDiffIndex {
+	void findMaxAtDiffIndex(int[] arr1, int[] arr2) {
+		int max = 0;
```

Comment:
No need to initialize here. You can directly initialize at line number 13.

Reply:
Hi @KrishnaKTechnocredits,
Made the suggested change.